nuget.config

```diff
@@ -4,6 +4,7 @@
     <clear />
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
     <add key="ProjectReunion-Dependencies" value="https://pkgs.dev.azure.com/ms/ProjectReunion/_packaging/ProjectReunion-Dependencies/nuget/v3/index.json" />
+    <add key="WindowsES-External" value="https://microsoft.pkgs.visualstudio.com/_packaging/WindowsES-External/nuget/v3/index.json" />
```

Comment:
I'm a bit fuzzy on the ES details, but I think any new NuGet feeds should be configured in the ProjectReunion-Dependencies pipeline. You should follow up with Reunion ES folks though.

Reply:
Also, IIRC you were explicitly asked not to use WindowsES-External.